doc/langref.html.in

```diff
@@ -3453,7 +3453,7 @@ test "aligned struct fields" {
           that variable.</li>
           <li>If the struct is in the {#syntax#}return{#endsyntax#} expression, it gets named after
           the function it is returning from, with the parameter values serialized.</li>
-          <li>Otherwise, the struct gets a name such as <code>(anonymous struct at file.zig:7:38)</code>.</li>
+          <li>Otherwise, the struct gets a name such as <code>struct:6:53</code>.</li>
```

Comment:
In stage2, which is expected to ship with 0.10.0, this will get a name like `test.test_0__struct_1024` so it would be better to use that directly instead of changing this again in a few months.